### ISSUE TITLE
"Convert to JUnit 4" not visible in Context Menu

### ISSUE THREAD

**Issue by kwin** [vogellacompany/codemodify#14]
I don't see a new entry "Convert to JUnit4" in the context menu for the package explorer on neither project, package or class level. This happens with a locally built Code Modifier 1.3.0 in Eclipse 2023-03 (Eclipse IDE for Enterprise Java and Web Developers).

<img width="427" alt="Screenshot 2023-10-18 at 10 22 18" src="https://github.com/vogellacompany/codemodify/assets/185025/ebaa79a2-fe07-4682-8a2d-24d26b9a774a">

**Comment by kwin** [comment]
Ok, meanwhile found it below "Source". That should be clarified in the readme.

**Comment by vogella** [comment]
Can you send a PR for the update of the readme?

Konrad Windszus ***@***.***> schrieb am Mi., 18. Okt. 2023,
10:26:

**Comment by kwin** [comment]
Yes, done in https://github.com/vogellacompany/codemodify/pull/15
